### DISCUSSION TITLE
typescriptreact prettier is not compatible with daisyUI?

### DISCUSSION THREAD

**Discussion by Papillon6814** [w0rp/ale#4254] [Q&A - Ask for help with problems] [answered]
Hello. I'm new to vim and ale. I'm trying to setup my vim like my ex-editor, vscode.
I used `prettier` for react development. So I'd like to use it in vim as well, and I added it with ale.

```
" Lint with ALE
let g:ale_fixers = {
\   '*': ['remove_trailing_lines', 'trim_whitespace'],
\   'typescriptreact': ['eslint'],
\   'typescript': ['prettier'],
\   'javascript': ['prettier'],
\   'javascriptreact': ['prettier'],
\   'css': ['prettier'],
\}

let g:ale_sign_error = 'P>'
let g:ale_sign_warning = 'P-'
let g:ale_echo_msg_format = '[%linter%] %code: %%s'
let g:ale_statusline_format = ['E%d', 'W%d', 'OK']

let g:ale_linters = {
\   'javascript': ['eslint'],
\}

let g:ale_fix_on_save = 1
let g:ale_completion_enabled = 1
let g:ale_javascript_prettier_use_local_config = 1
```

I added this setting in `~/.vimrc`.
And then I tried to edit my react project.

It should make my code prettier when I save it, but it generates surplus text at the top of a file.
```
[35m🌼 daisyUI components 2.19.0[0m [0m https://github.com/saadeghi/daisyui
[32m✔︎ Including:[0m [0m base, components, themes[13], utilities
```

I don't know why it is put and how to disable it. Does anyone got same issue as me?
Thank you.

**Comment by hsanson** [comment, marked as answer]
This is a problem with DaisyUI that is outputting via console.log things they shouldn't.  Refer to this discussion for more details and possible work around:

 - https://github.com/saadeghi/daisyui/issues/811#issuecomment-1133776027

**Reply by Papillon6814** [reply]
Thank you! It worked!
